### DISCUSSION TITLE
Is support for C planned?

### DISCUSSION THREAD

**Discussion by jfilby** [clockworklabs/SpacetimeDB#202] [Q&A]
I'm asking because I'd like to see support for Nim (https://nim-lang.org/). Nim transpiles to C.

I see that support for C++ is planned, so I would think that C would require less work.

**Comment by Mupu** [comment]
This would also open this up for way more languages, as basically every language knows how to talk to c modules. C++ not so much.

**Comment by L34Z** [comment]
I'd like to see C support as well, any chance for an answer on this?
